tasks/loadFixtures.js

```diff
@@ -0,0 +1,34 @@
+var _ = require('lodash');
+var request = require('request');
+var fs = require('fs');
+var path = require('path');
+var colors = require('ansicolors');
+
+module.exports = function (grunt) {
+  grunt.registerTask('loadFixtures', 'Loads fixtures into elasticsearch', function () {
+    const FIXTURES_PATH = path.join(grunt.config.get('root'), 'test/fixtures');
```

Comment:
Can we move this into config, or make it a bit more specific to the functional tests? At first I thought you were indexing [src/fixtures](https://github.com/elastic/kibana/tree/master/src/fixtures)